src/main/java/org/asamk/signal/commands/ListGroupsCommand.java

```diff
@@ -9,32 +9,61 @@
 import org.asamk.signal.manager.storage.groups.GroupInfo;
 import org.whispersystems.signalservice.api.push.SignalServiceAddress;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 public class ListGroupsCommand implements LocalCommand {
 
-    private static void printGroup(Manager m, GroupInfo group, boolean detailed) {
-        if (detailed) {
-            Set<String> members = group.getMembers()
-                    .stream()
-                    .map(m::resolveSignalServiceAddress)
-                    .map(SignalServiceAddress::getLegacyIdentifier)
-                    .collect(Collectors.toSet());
-
-            Set<String> pendingMembers = group.getPendingMembers()
-                    .stream()
-                    .map(m::resolveSignalServiceAddress)
-                    .map(SignalServiceAddress::getLegacyIdentifier)
-                    .collect(Collectors.toSet());
-
-            Set<String> requestingMembers = group.getRequestingMembers()
-                    .stream()
-                    .map(m::resolveSignalServiceAddress)
-                    .map(SignalServiceAddress::getLegacyIdentifier)
-                    .collect(Collectors.toSet());
+    private enum MembersType {
```

Comment:
I don't like creating an enum just for the helper method, how about extracting a `resolveMembers` method and using that below.
```
Set<String> resolveMembers(Manager m, Set<SignalServiceAddress> addresses)

resolveMembers(m, group.getMembers())
resolveMembers(m, group.getPendingMembers())
resolveMembers(m, group.getRequestingMembers())
```